docs/hub/repositories-getting-started.md

```diff
@@ -132,6 +132,9 @@ And you're done! You can check your repository on Hugging Face with all the rece
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/repo_with_files-dark.png"/>
 </div>
 
+<Tip>
+You might find it useful to setup a [git credential helper](https://git-scm.com/docs/gitcredentials) to avoid filling your username and password on every push operation.
```

Comment:
i know its weird but i think GitHub's doc about this is better than https://git-scm.com/docs/gitcredentials

Or maybe better, do a short version of GitHub's doc in our doc

Reply:
hmmm, the only thing I find seems to be specific to GitHub: https://docs.github.com/en/get-started/getting-started-with-git/caching-your-github-credentials-in-git. Maybe they changed recently (_Password-based authentication for Git has been removed in favor of more secure authentication methods_)

Reply:
then maybe a super short extract from the git doc

BTW you also need to mention and link to the git+ssh doc, cc @SBrandeis (it's probably the simplest for anyone having trouble with setting up git-credential)

Reply:
I changed a bit, hopefully it's enough (super short extract from the git doc -> I think we should not push that solution, and it depends on every user OS / configuration, so... better to let them figure how to set it up)